plugin-server/src/types.ts

```diff
@@ -150,6 +150,7 @@ export interface PluginsServerConfig extends Record<string, any> {
     OBJECT_STORAGE_SECRET_ACCESS_KEY: string
     OBJECT_STORAGE_SESSION_RECORDING_FOLDER: string
     OBJECT_STORAGE_BUCKET: string
+    OBJECT_STORAGE_TEAM_ALLOW_LIST: 'all' | number[]
```

Comment:
Suggestion: Let's continue the same pattern as with other env vars rather than introducing a new one. So remove 'all' and make this a string same as `CONVERSION_BUFFER_ENABLED_TEAMS`. If we want to turn this on for everyone, let's yeet the code or add a separate var.